src/tests/multihost/ipa/conftest.py

```diff
@@ -165,6 +166,80 @@ def restoresssdconf():
 # ====================  Class Scoped Fixtures ================
 
 
+@pytest.fixture(scope='class')
+def environment_setup(session_multihost, request):
+    """
+    Install necessary packages
+    """
+    session_multihost.client[0].run_command("yum "
+                                            "--enablerepo=rhel-CRB install"
```

Comment:
rhel-CRB repo is not available in other OSes. It will help to add logic to either detect the OS or enable the CRB repo only if package install fails.

Reply:
now passed with IDM-CI : https://ci-jenkins-csb-idmops.apps.ocp-c1.prod.psi.redhat.com/job/trigger-test-suite-tool/536/console